core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -72,6 +72,10 @@ oppia.factory('AssetsBackendApiService', [
     };
 
     var _getAudioDownloadUrl = function(explorationId, filename) {
+      if (GLOBALS.DEV_MODE) {
+        return '/assets/test/' + filename;
```

Comment:
Not really sure how to go about doing the latter implementation while still avoiding an if/else for the url interpolation?